keepassxc-browser/common/sites.js

```diff
@@ -97,6 +98,25 @@ kpxcSites.exceptionFound = function(identifier) {
     return false;
 };
 
+/**
+ * Handles a few exceptions for certain sites where 2FA field is not regognized properly.
+ * @param {object} field   Input field Element
+ * @returns {boolean}      True if an Element has a match with the needed indentfifiers and document location
+ */
+kpxcSites.totpExceptionFound = function(field) {
+    if (!field || field.nodeName !== 'INPUT') {
+        return false;
+    }
+
+    if (document.location.href === 'https://twitter.com/i/flow/login'
+        && field.autocomplete === 'on' && field.dir === 'auto'
+        && field.name === 'text' && field.type === 'text') {
```

Comment:
Why is it so hard for these clowns to just call their totp field totp or otp or something useful?

Reply:
Dunno. Gonna give Twitter some feedback. Please be friendly to password managers in the future.